chainstate/storage/src/lib.rs

```diff
@@ -258,4 +258,4 @@ pub trait Transactional<'t> {
     fn transaction_rw<'s: 't>(&'s self, size: Option<usize>) -> Result<Self::TransactionRw>;
 }
 
-pub trait BlockchainStorage: BlockchainStorageWrite + for<'tx> Transactional<'tx> + Send {}
```

Comment:
There is the `IsTransaction` marker trait, defined further up in this file. I believe it's used to differentiate storage access from transaction and storage access directly. Since the latter no longer exists, I think it can be removed. Unless it's also used for something else...

Reply:
Done